pages/authorDashboard/PKPAuthorDashboardHandler.php

```diff
@@ -302,8 +304,8 @@ public function setupTemplate($request)
         $state = [
             'canEditPublication' => $canEditPublication,
             'components' => [
-                FORM_TITLE_ABSTRACT => $titleAbstractForm->getConfig(),
-                FORM_CITATIONS => $citationsForm->getConfig(),
+                FORM_TITLE_ABSTRACT => $this->getLocalizedForm($titleAbstractForm, $submission, $submissionContext),
```

Comment:
L165-168:
`$submissionContext = $request->getContext();`
`        if ($submission->getContextId() !== $submissionContext->getId()) {`
`            $submissionContext = Services::get('context')->get($submission->getContextId());`
`        }`
I used it because of above if-statement. If you think that getLocalizedForm should do this check then I move it there (?)

Reply:
Hmmm... I do not know why that if is there -- the submission context should not be different from the request context... :thinking: ... accessing a submission from another context does/should not work.... :thinking: 
But... because the code is pretty old... lets keep it that way... here and below...
Thanks for explanation! :-)